server/src/db/index.ts

```diff
@@ -0,0 +1,48 @@
+import mongoose, {Schema} from 'mongoose';
+
+// Recruiter accounts
+let UserSchema : Schema = new Schema({
+  name: String,
+  email: String,
+  password: String
+})
+
+// Questions
+let QuestionSchema : Schema = new Schema({
+  prompt: String,
+  options: [String],
+  type: String
+})
+
+// Answers
+// TODO should this be a separate schema or embedded?
+let AnswerSchema : Schema  = new Schema({
+  question: {type: mongoose.Schema.Types.ObjectId, ref: 'Question'},
+  value: String
+})
+
+// Response
+let ResponseSchema : Schema = new Schema({
+  respondent: String,
+  answers: [{ type : mongoose.Schema.Types.ObjectId, ref: 'Answer' }],
+  scoring: {
+    score: Number,
+    notes: String
+  },
+  timestamps: true
+})
+
+let InterviewSlotSchema: Schema = new Schema({
+  start: Date,
+  end: Date,
+  available: Boolean,
+  email: String
```

Comment:
`intervieweeEmail` might be more clear, or maybe a reference to the `Response` object

Reply:
Will leave it as an email for now but can change later if needed.